samples/interactive-tutorials/src/main/java/events/setup/EventsCreateBigQueryTable.java

```diff
@@ -30,16 +30,15 @@
 
 public class EventsCreateBigQueryTable {
 
-  public static void main(String... args) throws IOException {
+  public static void main(String[] args) throws IOException {
     String dataset = "user_events";
     String validEventsTable = "events";
     String invalidEventsTable = "events_some_invalid";
     String eventsSchemaFilePath = "src/main/resources/events_schema.json";
     String validEventsSourceFile =
-        String.format("gs://%s/user_events.json", EventsCreateGcsBucket.getBucketName());
+        String.format("gs://%s/user_events.json", System.getenv("EVENTS_BUCKET_NAME"));
```

Comment:
Can we move the `EVENTS_BUCKET_NAME` as a static final constant and reuse it the both places here?